cmd/util/cmd/execution-state-extract/cmd.go

```diff
@@ -137,10 +134,10 @@ func run(*cobra.Command, []string) {
 		}
 	}
 
-	log.Info().Msgf("Extracting state from %s, exporting root checkpoint to %s, version: %v",
+	log.Info().Msgf("Extracting state from %s, exporting root checkpoint to %s, version: 6",
```

Comment:
nit: maybe use `wal.MaxVersion` instead of hard-coded version `6` assuming that we only extract checkpoint file in latest version.